donut/modules/courses/templates/scheduler.html

```diff
@@ -0,0 +1,424 @@
+{% extends 'layout.html' %}
```

Comment:
General comments:
- The check box for the sections are spaced and misaligned from the section number.
- Is it possible to make it so when you click in the box the check box is checked/unchecked? Also, for the pop up for the sections, if you uncheck the check box, the pop up disappears. I think it'd be better if it stayed, especially if there's no way to make the box check/uncheck the check box.
- The last hour is a cut off (or i guess it shouldn't be shown).
- When beginning a search, you probably want to reset the scroll bar to the top.
- Again, mobile behavior is somewhat strange (I just made the window very small, and you can see).
- If you open scheduler in two different tabs, and they both try to add the same course, the error that pops up on the second one is "Failed to drop course..."